service/controller/cloudconfig/cloud_config.go

```diff
@@ -78,6 +80,7 @@ func New(config Config) (*CloudConfig, error) {
 
 		azure:                  config.Azure,
 		azureClientCredentials: config.AzureClientCredentials,
+		dockerhubToken:         config.DockerhubToken,
```

Comment:
should we check if the token is empty and fail in that case?

Reply:
yes, added